darwin/torch/dataset.py

```diff
@@ -310,12 +310,13 @@ def get_target(self, index: int) -> Dict[str, Any]:
 
         annotations = []
         for annotation in target["annotations"]:
-            if "polygon" not in annotation and "complex_polygon" not in annotation:
+            annotation_type: str = annotation.annotation_class.annotation_type
+            path_key = "paths" if annotation_type == "complex_polygon" else "path"
+            if path_key not in annotation.data:
                 print(f"Warning: missing polygon in annotation {self.annotations_path[index]}")
```

Comment:
It's not your code, but it would help our bug diagnosis if this used a logger.  If it's too much for a SWAT ticket, hit me back and I'll add it to my backlog.

Reply:
I'd rather change as little as possible if that's ok? Especially since I'm not really a python expert.